adoc/admin-stratos-web-console.adoc

```diff
@@ -0,0 +1,233 @@
+= Stratos Web Console
+
+The Stratos user interface (UI) is a modern web-based management application for
+{kube} and for Cloud Foundry distributions based on {kube} like SUSE
+Cloud Application Platform.
+
+Stratos provides a graphical management console for both developers and system
+administrators.
+
+A single Stratos instance can be used to monitor multiple {kube} clusters
+as long as it is granted access to their {kube} API endpoint.
+
+If you plan to deploy SUSE Cloud Application Platform on your {productname}
+cluster please skip this section of the documentation and refer
+to the official SUSE Cloud Application Platform instructions.
+
+== Introduction
+
+This document aims to describe how to install Stratos in a {productname} cluster
+that doesn't plan to run any SUSE Cloud Application Platform components.
+
+The Stratos stack is deployed using a helm charts and consists of its web
+UI POD and a MariaDB one that is used to store configuration values.
+
+== Prerequisites
```

Comment:
Maybe we could also mention helm and tiller.
Here is what's on CAP doc.
`Helm's Tiller has to be installed and active, with Tiller on the Kubernetes cluster and Helm on your remote administration machine`

Reply:
Done, thanks